app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlComponent.kt

```diff
@@ -148,38 +150,44 @@ fun Observer<SessionControlAction>.onNext(onboardingAction: OnboardingAction) {
 sealed class SessionControlChange : Change {
     data class Change(val tabs: List<Tab>, val mode: Mode, val collections: List<TabCollection>) :
         SessionControlChange()
+
     data class TabsChange(val tabs: List<Tab>) : SessionControlChange()
-    data class ModeChange(val mode: Mode) : SessionControlChange()
+    data class ModeChange(val mode: Mode, val tabs: List<Tab>? = null) : SessionControlChange()
```

Comment:
This is the only nonformatting change. Pass new list of tabs into Mode Change action